types/postcss-prefix-selector/index.d.ts

```diff
@@ -1,16 +1,16 @@
-// Type definitions for postcss-prefix-selector 1.14
+// Type definitions for postcss-prefix-selector 1.15
 // Project: https://github.com/RadValentin/postcss-prefix-selector
 // Definitions by: robertmaier <https://github.com/robertmaier>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/postcss-prefix-selector
 
 interface Options {
-    prefix?: string;
-    exclude?: ReadonlyArray<string | RegExp>;
-    ignoreFiles?: ReadonlyArray<string | RegExp>;
-    includeFiles?: ReadonlyArray<string | RegExp>;
-    transform?: (prefix: Readonly<string>, selector: Readonly<string>, prefixedSelector?: Readonly<string>, file?: Readonly<string>) => string;
+    prefix: string;
```

Comment:
```suggestion
    prefix?: string | undefined;
```
I'm really sorry, I was wrong, this is not required option, I've misread. Sorry for invenvienance.